shell/platform/linux/fl_renderer_gl.cc

```diff
@@ -109,7 +110,28 @@ static gboolean fl_renderer_gl_present_layers(FlRenderer* renderer,
             reinterpret_cast<FlBackingStoreProvider*>(framebuffer->user_data));
       } break;
       case kFlutterLayerContentTypePlatformView: {
-        // Currently unsupported.
+        FlPlatformViewsPlugin* plugin = fl_view_get_platform_views_plugin(view);
+        FlPlatformView* platform_view =
+            fl_platform_views_plugin_get_platform_view(
+                plugin, layer->platform_view->identifier);
+        GtkWidget* widget = fl_platform_view_get_view(platform_view);
+        if (!widget)
+          continue;
+        GdkRectangle geometry = {
+            .x = static_cast<int>(layer->offset.x),
+            .y = static_cast<int>(layer->offset.y),
+            .width = static_cast<int>(layer->size.width),
+            .height = static_cast<int>(layer->size.height),
+        };
+        GPtrArray* mutations =
```

Comment:
Use `g_autoptr` here once the ownership is changed on `fl_view_add_widget`.

Reply:
Done.